public/apps/configuration/panels/get-started.tsx

```diff
@@ -188,7 +188,6 @@ export function GetStarted(props: AppDependencies) {
 
```

Comment:
Revert this change

Reply:
complete by rewriting the changes to this branch (with the changes from main & the edit to the button needed to close this ticket).